docs/modules/ROOT/pages/configuration.adoc

```diff
@@ -452,8 +452,12 @@ Most cops are enabled by default. Cops, introduced or significantly updated
 between major versions, are in a special pending status (read more in
 xref:versioning.adoc["Versioning"]). Some cops, configured the above `Enabled: false`
 in https://github.com/rubocop/rubocop/blob/master/config/default.yml[config/default.yml],
-are disabled by default. The cop enabling process can be altered by
-setting `DisabledByDefault` or `EnabledByDefault` (but not both) to `true`.
+are disabled by default.
+
+The cop enabling process can be altered by setting `DisabledByDefault` or
+`EnabledByDefault` (but not both) to `true`. These settings override the default *all*
```

Comment:
the default or by default? Something definitely seems wrong about this sentence. :-)

Reply:
🤦 thanks for catching! I've updated it.